tools/codegen/api/lazy.py

```diff
@@ -39,6 +39,8 @@ def process_ir_type(typ: Type) -> Union[BaseCType, VectorCType, OptionalCType, L
             return BaseCType(scalarTypeT)
         elif typ.name == BaseTy.int:
             return BaseCType(longT)
+        elif typ.name == BaseTy.SymInt:
+            return BaseCType(valueT)
```

Comment:
unrelated to your PR: should we name this lazyIrValueT or lazyValueT at least?  I think i picked the most confusing-possible name here

Reply:
frankly, I don't find `valueT` particularly confusing. the confusing part is that it isn't a real Pytorch type. Maybe lazyIrValueT would hint at that.